docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,382 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
 $ sensorctl test accelerometer 100 1000 0 /* enable accelerometer with interval 100 ms, 1s batch latency and always on option */
 $ sensorctl info accelerometer /* retrieve accelerometer sensor information */
 ```
+
+## Crash Framework
+
+### crash-manager
+
+crash-manager is a tool used to create crash reports when an application gets crashed.
+crash-manager is not invoked manually, it is invoked automatically by the kernel when the crash occurs. crash-manager generates the crash report and attaches the kernel-provided core to it, if requested. crash-worker package provides the `99-crash-manager.conf` file, which configures kernel parameters for crash-manager to get invoked automatically on crash. For more information, see `99-crash-manager.conf`.
+
+crash-manager consists of the following options:
+
+Option                | Description
+----------------------|------------------------------------------------------
+-p  --pid=PID         | Specifies the process ID (PID) of the dumped process
+-u  --uid=UID         | Specifies the real user identifier (UID) of the dumped process
+-g  --gid=GID         | Specifies the real group identifier (GID) of the dumped process
+-i  --tid=TID         | Specifies the thread identifier (TID) of the thread that triggered core dump. For more information, see [core(5)](http://man7.org/linux/man-pages/man5/core.5.html).
+-s  --signal=SIG      | Specifies the signal number causing dump
+-t  --time=TIME       | Specifies the time of dump in seconds since Epoch
```

Comment:
@mijin85cho,

Thank you for the explanation. As per my understanding I took Epoch as a proper nounce, hence I suggested to remove the article "the".

@mmoscicki2,
Please rephrase to: Specifies the time of dump in seconds since the epoch.